src/getOperands.js

```diff
@@ -26,7 +26,9 @@
  * @return {string} - operands
  */
 function getOperands(str) {
-  // write code here
+  let check = /[+-]?([0-9]*[.])?[0-9]+/g;
```

Comment:
name should explain what this regular expression means. 'check' is way too generic for a name.